src/transactions/typeTag/typeTag.ts

```diff
@@ -45,9 +45,63 @@ export abstract class TypeTag extends Serializable {
         throw new Error(`Unknown variant index for TypeTag: ${index}`);
     }
   }
+
```

Comment:
rename file? typeTag/typeTag is a bit weird

Reply:
True... probably just move it to index